package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "extensions-official",
+  "name": "extensions-sources",
   "version": "1.0.0",
```

Since it's a new branch, would you mind adding a `repositoryName` field?

Something like:
```suggestion
  "name": "extensions-sources",
  "repositoryName": "H-Extensions",
  "version": "1.0.0",
```

---

Done in 383bfd8eff45b7fcc4f2ceb9af9a62b1c8aa13b3